Patch-Source: https://gitlab.gnome.org/GNOME/gimp/-/commit/11892f1d83ffc465346dab7e2e8c6e790f555a64
--
From 11892f1d83ffc465346dab7e2e8c6e790f555a64 Mon Sep 17 00:00:00 2001
From: Jehan <jehan@girinstud.io>
Date: Wed, 21 Feb 2024 11:08:23 +0100
Subject: [PATCH] Issue #10900: install missing gimpchoice.h.

Argh how this could pass without anyone noticing?!
---
 libgimpbase/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libgimpbase/meson.build b/libgimpbase/meson.build
index a77d125f39d..a2c27016b10 100644
--- a/libgimpbase/meson.build
+++ b/libgimpbase/meson.build
@@ -84,6 +84,7 @@ libgimpbase_headers_introspectable = files(
   'gimpbaseenums.h',
   'gimpbasetypes.h',
   'gimpchecks.h',
+  'gimpchoice.h',
   'gimpcpuaccel.h',
   'gimpenv.h',
   'gimplimits.h',
-- 
GitLab

